aperturedb/ParallelQuery.py

```diff
@@ -182,7 +183,7 @@ def process_responses(requests, input_blobs, responses, output_blobs):
                 response_handler = self.generator.response_handler
             if hasattr(self.generator, "strict_response_validation") and isinstance(self.generator.strict_response_validation, bool):
                 strict_response_validation = self.generator.strict_response_validation
-            result, r, b = execute_batch(
+            result, r, b = self.batch_command(
```

Comment:
we replace direct call of execute_batch to allow PQS insert its call and utilize the PQ generate_batch